### PR TITLE
Add extended Doctrine AnnotationReader

### PR DESCRIPTION
**Pull request by niels-nijens**
This PR adds an extended Doctrine AnnotationReader to be able to return all annotation instances of all properties tagged with the specified annotation.

### CODE REVIEW

**Comment by niels-nijens** [comment]
Waiting for #2 to be finished.

File: Tests/AnnotatedMock.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+     */
+    private $qux = array();
+
+    /**
+     * @GDPR\Export()
+     *
+     * @var ArrayCollection
+     */
+    private $quux;
+
```
Comment (hansgoed) [inline Tests/AnnotatedMock.php]: Can you add a test for fields that aren't annotated?

Comment (niels-nijens) [reply Tests/AnnotatedMock.php]: Added the `notAnnotatedProperty` property. 😁

File: Tests/AnnotatedMock.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    /**
+     * The quux property.
+     *
+     * @GDPR\Export()
+     *
+     * @var ArrayCollection
+     */
+    private $quux;
+
```
[the remark this answers is not in the document]
Comment (hansgoed) [reply Tests/AnnotatedMock.php]: You are absolutely right.

**Pull request merged**